src/figuresClasses.ts

```diff
@@ -1,19 +1,77 @@
 export interface Figure {
-
+  shape: 'triangle' | 'circle' | 'rectangle',
+  color: 'red' | 'green' | 'blue',
+  getArea: () => number,
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: 'triangle' = 'triangle';
 
-}
+  constructor(
+    public color: 'red' | 'green' | 'blue',
+    public sideA: number,
+    public sideB: number,
+    public sideC: number,
+  ) {
+    if (
+      sideA >= sideB + sideC
+      || sideB >= sideA + sideC
+      || sideC >= sideA + sideB
+      || sideA <= 0
+      || sideB <= 0
+      || sideC <= 0
+    ) {
+      throw new Error('Invalid triangle');
+    }
+  }
```

Comment:
maybe it would be better to do it with sort method? too much code, check the solution which Sergiy published in Slack, if u need some help to understand